src/Api/HL/Controller/ITILController.php

```diff
@@ -170,6 +177,122 @@ public static function getRawKnownSchemas(): array
             ]
         ];
 
+        $schemas['RecurringTicket'] = [
+            'x-itemtype' => \TicketRecurrent::class,
+            'type' => Doc\Schema::TYPE_OBJECT,
+            'description' => 'Recurring ticket',
+            'properties' => [
+                'id' => [
+                    'type' => Doc\Schema::TYPE_INTEGER,
+                    'format' => Doc\Schema::FORMAT_INTEGER_INT64,
+                    'x-readonly' => true,
+                ],
+                'name' => ['type' => Doc\Schema::TYPE_STRING],
+                'comment' => ['type' => Doc\Schema::TYPE_STRING],
+                'entity' => self::getDropdownTypeSchema(Entity::class),
+                'is_recursive' => ['type' => Doc\Schema::TYPE_BOOLEAN],
+                'is_active' => ['type' => Doc\Schema::TYPE_BOOLEAN],
+                'template' => self::getDropdownTypeSchema(TicketTemplate::class),
+                'date_begin' => [
```

Comment:
It is. Most date properties seemed to be `date_*` so that was what I standardized on. It can be switched to standardize on names that match with English better.